cmd/quadlet/main.go

```diff
@@ -199,9 +199,10 @@ func nonNumericFilter(_path string, isUserFlag bool) bool {
 }
 
 func userLevelFilter(_path string, isUserFlag bool) bool {
+	resolvedUnitDirAdmin, _ := filepath.EvalSymlinks(quadlet.UnitDirAdmin)
```

Comment:
Should check for error here and logrus.Errorf it.
And then set 
resolvedUnitDirAdmin=quadlet.UnitDirAdmin  to make sure it has the original value we used later.

I think the resolvedUnitDirAdmin will equal "" if an error is thrown.

Reply:
fixed. I had previously ignored the case that the path may not exist.